machine-runner/src/utils/type-utils.ts

```diff
@@ -23,12 +23,12 @@ export type NotAnyOrUnknown<T> = any extends T ? never : T
 
 type SerializablePrimitive = number | string | boolean | null | undefined
 
+type SerializableRecord = Record<string, SerializablePrimitive>
```

Comment:
Does this mean that `{ [_: string]: Record<string, number[]> }` is not considered serialisable? Or is that covered via SerializableObject?

Reply:
You're right. It should have been SerializableObject here.

Reply:
Fixed in https://github.com/Actyx/machines/commit/4e14c47119e5bfaffb3479731fb8241b2d7dffef